scripts/publish-docs.sh

```diff
@@ -94,8 +93,10 @@ fi
 git push docs gh-pages
 popd
 
-rm -rf gh-pages
-git worktree prune
+# Cleanup gh-pages state for future runs based on remote
+git worktree remove gh-pages
+# above takes care of removal: rm -rf gh-pages
+git branch -D gh-pages
```

Comment:
Could you please provide more context why it should be changed and how does it change existing behavior? Not a `worktree` master :)
Just don't understand how does it aligns with `git worktree add gh-pages gh-pages` in line 69.

Reply:
While attempting to publish and fix the bugs my repo got wedged multiple times, requiring to do these manual steps as instructed in our docs. This way we avoid unpredictable state and just start clean from wherever we publish.